### PR TITLE
Add 3 point angle annotation type

### PR DESCRIPTION
**Pull request by pkcakeout**
- Add json schema(s)
- Add test data
- Add migrations

Pitch https://github.com/DIAGNijmegen/rse-roadmap/issues/283

### CODE REVIEW

File: app/grandchallenge/components/schemas.py [hunk 1]
```diff
@@ -491,6 +491,67 @@
...
+            },
+            "required": ["angle"],
+            "additionalProperties": False,
+        },
+        "M3AN": {
+            "type": "object",
+            "properties": {
+                "name": {"type": "string"},
+                "type": {"enum": ["Multiple three-point angles"]},
+                "boxes": {
```
Comment (jmsmkn) [inline app/grandchallenge/components/schemas.py:542]: Wouldn't `"angles"` be more appropriate than `"boxes"` here?

File: app/grandchallenge/components/schemas.py [hunk 2]
```diff
@@ -491,6 +491,67 @@
...
+                        "minItems": 3,
+                        "maxItems": 3,
+                    },
+                    "minItems": 3,
+                    "maxItems": 3,
+                },
+                "version": {"$ref": "#/definitions/version-object"},
+                "probability": {"type": "number", "minimum": 0, "maximum": 1},
+            },
+            "required": ["angle"],
```
Comment (jmsmkn) [inline app/grandchallenge/components/schemas.py:534]: Shouldn't `"version"` also be required?

File: app/tests/components_tests/test_serializers.py [hunk 3]
```diff
@@ -195,6 +195,45 @@
...
+            [76.801, -57.62, 84.43],
+            [77.10, -18.97, 115.48],
+        ],
+        "probability": 0.92,
+        "version": {"major": 1, "minor": 0},
+    },
+    "M3AN": {
+        "name": "Some annotations",
+        "type": "Multiple three-point angles",
+        "angles": [
```
Comment (jmsmkn) [inline app/tests/components_tests/test_serializers.py:212]: How did this test pass?

File: app/grandchallenge/components/schemas.py (hunk 1 above)
Comment (pkcakeout) [reply app/grandchallenge/components/schemas.py:542]: Sorry yes! Too many updates necessary, this is a copy-paste mistake!

File: app/grandchallenge/components/schemas.py (hunk 2 above)
Comment (pkcakeout) [reply app/grandchallenge/components/schemas.py:534]: Reviewing this myself yields that you are right! For the "3ANG", this should be required. The "xxx-object" types do not have those... fixing!

Comment (jmsmkn) [reply app/grandchallenge/components/schemas.py:534]: > The "xxx-object" types do not have those... 

I think that is correct, they're only used in the Multi-x-object, the versioning on those is done at the top level.

File: app/tests/components_tests/test_serializers.py (hunk 3 above)
Comment (pkcakeout) [reply app/tests/components_tests/test_serializers.py:213]: I forgot to include the 3-angle types in an enumeration that happened for the pytest-parameterized tests... that was the problem. Fixed!

File: app/grandchallenge/reader_studies/models.py [hunk 4]
```diff
@@ -1204,6 +1216,8 @@ class QuestionWidgetKindChoices(models.TextChoices):
     AnswerType.MULTIPLE_ELLIPSES: None,
     AnswerType.MULTIPLE_CHOICE: [],
     AnswerType.MULTIPLE_CHOICE_DROPDOWN: [],
+    AnswerType.THREE_POINT_ANGLE: None,
+    AnswerType.MULTIPLE_THREE_POINT_ANGLES: None,
```
[the remark this answers is not in the document]
Comment (pkcakeout) [reply app/grandchallenge/reader_studies/models.py:1220]: Added 2fbad11a372114af2a369d094a5e75e78481f6e0 - I assume that is what you maybe meant for the reader-study tests? Thanks for the hint!

**Review by jmsmkn — APPROVED** [review]

**Pull request merged**